src/Appwrite/Platform/Modules/Databases/Http/Databases/Create.php

```diff
@@ -80,13 +80,14 @@ public function __construct()
             ->param('databaseId', '', new CustomId(), 'Unique Id. Choose a custom ID or generate a random ID with `ID.unique()`. Valid chars are a-z, A-Z, 0-9, period, hyphen, and underscore. Can\'t start with a special char. Max length is 36 chars.')
             ->param('name', '', new Text(128), 'Database name. Max length: 128 chars.')
             ->param('enabled', true, new Boolean(), 'Is the database enabled? When set to \'disabled\', users cannot access the database but Server SDKs with an API key can still read and write to the database. No data is lost when this is toggled.', true)
+            ->param('type', 'mysql', new Text(128), 'Database type.', true)
```

Comment:
The validator here should be a `Whitelist` of `sql` and `nosql` with default sql (changed from mysql)